README.md

```diff
@@ -60,6 +60,23 @@ inherit_gem:
 # your customizations here...
 ```
 
+### Custom Cops
+
+To activate the custom cops (`mdsol/*`) shipped with this Gem:
+
+```yaml
+require: rubocop-performance
```

Comment:
not sure I understand this example, does it mean to be able to use the custom cops we need rubocop-performance ??

Reply:
That was a terrible copy-paste mistake... 8a00579 🙇